Developers/Newcomers_Guide.rst

```diff
@@ -52,7 +52,7 @@ invitation to our org. This is your first step towards contributing.
 A maintainer will command ``cobot`` (our gitter bot) to invite
 you and be part of the Newcomer team.
```

Comment:
the commit shortlog is not that helpful: You are saying you are fixing links... when you arent.

"Improve link display names" is way better